glmnet/logistic.py

```diff
@@ -148,7 +148,7 @@ def __init__(self, alpha=1, n_lambda=100, min_lambda_ratio=1e-4,
 
         self.cv = None
 
-    def fit(self, X, y, sample_weight=None, relative_penalties=None):
+    def fit(self, X, y, sample_weight=None, relative_penalties=None, max_features=None):
```

Comment:
According to the scikit-learn API, `max_features` is probably not a fit parameter so much as an `__init__` parameter; so we should move it to the `__init__` function of both classes.

scikit-learn convention aside, doing so would remove it from the function signature of this function, `_fit`, and `_fit_and_score` because `max_features` would just be on the class.